.travis.yml

```diff
@@ -16,8 +16,10 @@ script:
 - make check
 - make test
 - make build
+- make clean
 
 deploy:
+  skip_cleanup: true
```

Comment:
Why do you want to skip cleanup?

Reply:
You are actually right. We don't need to skip clean since `make push` will automatically trigger all other make actions to build the app and image. I've removed that line.